.storybook/preview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "metabase/css/index.css";
+import { MantineProvider } from "@mantine/core";
+import { theme } from "../frontend/src/metabase/ui/theme";
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
+
+function ThemeWrapper(props) {
+  return (
+    <MantineProvider theme={theme} withNormalizeCSS>
```

Comment:
I wonder if we should create a wrapped version of this so that we share the properties like `withNormalizeCSS` across Storybook and `app.js`?

Reply:
yeah I would probably create our own `ThemeProvider` and wrap `MantineProvider`, so we could use `<ThemeProvider>...</ThemeProvider>` without knowing inner details